ext/date/date_core.c

```diff
@@ -5640,12 +5645,17 @@ d_lite_leap_p(VALUE self)
 
 /*
  * call-seq:
- *    d.start  ->  float
+ *   start -> float or Infinity or -Infinity
+ *
+ * Returns the start date for calendar reform:
```

Comment:
I think we want to mention that the returned value is the day in the Julian calendar (i.e. we can pass the returned value to `Date.jd` to get a `Date`).

Reply:
Fixed.